config/manager/controller_manager_config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+kind: ControllerManagerConfig
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: 127.0.0.1:8080
```

Comment:
Why is one bound to localhost/127 whereas the healthProbe appears to be wild card?

Reply:
Since this is operator-sdk created and managed content, documentation is rather sparse on why this is. I believe it is because metrics is only needed to be accessed internally to the cluster whereas the health probe needs to be binded to localhost and the IP given to the pod via the cluster.